apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.ts

```diff
@@ -81,7 +121,7 @@ router.post(
         res.locals.plainUrlPrefix +
           '/course_instance/' +
           courseInstanceId +
-          '/instructor/instance_admin/settings',
+          '/instructor/instance_admin/assessments',
```

Comment:
Consider adding a flash message here to let users know that everything went ok. You can see an example of that here:

https://github.com/PrairieLearn/PrairieLearn/blob/b8cb11c28489bdad4b0e7d6b18ed6939442d4e93/apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.ts#L171

Consider text like "Course instance created successfully."

Reply:
I added this:

<img width="1700" alt="image" src="https://github.com/user-attachments/assets/05ed03fe-7558-4695-992f-b5420c99cc1d" />

Reply:
Nice! Can you add a period at the end? I know the example I linked to doesn't include one, but I think that's an oversight.

Reply:
Yes! I added this change:

<img width="870" alt="image" src="https://github.com/user-attachments/assets/51cee244-cf36-4633-ac17-c5bb6d550a30" />

Reply:
Perfect!